docs/definitions/release_summary_with_checks.csv

```diff
@@ -1,55 +1,55 @@
+first_contract_datesigned,timestamp,First `dateSigned` across all contracts
+last_contract_datesigned,timestamp,Last `dateSigned` across all contracts
+total_contract_documents,numeric,Count of contracts/documents
+total_contract_milestones,numeric,Count of contracts/milestones
+total_contract_items,numeric,Count of contracts/items
+total_contract_implementation_documents,numeric,Count of contracts/implementation/documents
+total_contract_implementation_milestones,numeric,Count of contracts/implementation/milestones
+contract_documenttype_counts,jsonb,JSONB object with the keys as unique contracts/documents/documentType and the values as count of the appearances of those documentTypes
+contract_implemetation_documenttype_counts,jsonb,JSONB object with the keys as unique contracts/implementation/documents/documentType and the values as count of the appearances of those documentTypes
+contract_milestonetype_counts,jsonb,JSONB object with the keys as unique contracts/milestone/milestoneType and the values as count of the appearances of those milestoneTypes
+contract_implementation_milestonetype_counts,jsonb,JSONB object with the keys as unique contracts/implementation/documents/milestoneType and the values as count of the appearances of those milestoneType
+total_documenttype_counts,jsonb,JSONB object with the keys as unique documentTypes from all documents in the release and the values as count of the appearances of those documentType
+total_documents,numeric,Count of documents in the release
+milestonetype_counts,jsonb,JSONB object with the keys as unique milestoneTypes from all milestones in the release and the values as count of the appearances of those milestoneType
+total_milestones,numeric,Count of milestones in the release
+source_id,text,`source_id` from kingfisher process collection table
+data_version,timestamp,`data_version` from kingfisher process collection table
+store_start_at,timestamp,`store_start_at` from kingfisher process collection table
+store_end_at,timestamp,`store_end_at` from kingfisher process collection table
+sample,boolean,`sample` from kingfisher process collection table
+transform_type,text,`transform_type` from kingfisher process collection table
+transform_from_collection_id,integer,`transform_from_collection_id` from kingfisher process collection table
+deleted_at,timestamp,`deleted_at` from kingfisher process collection table
+release_check,jsonb,JSONB of Data Review Tool output which includes validation errors and additional field information
+release_check11,jsonb,JSONB of Data Review Tool output run against 11 version of OCDS even if the data is from 10
+record_check,jsonb,JSONB of Data Review Tool output which includes validation errors and additional field information
+record_check11,jsonb,JSONB of Data Review Tool output run against 11 version of OCDS even if the data is from 10
```

Comment:
JSONB of Data Review Tool output run against 11 version of OCDS even if the data is from 10 -> JSONB of Data Review Tool output run against  OCDS version 1.1 even if the data is from OCDS version 1.0